components/layout_2020/flow/root.rs

```diff
@@ -446,11 +448,15 @@ impl FragmentTree {
         })
     }
 
-    pub fn remove_nodes_in_fragment_tree_from_set(&self, set: &mut FxHashSet<OpaqueNode>) {
+    pub fn remove_nodes_in_fragment_tree_from_set(&self, set: &mut FxHashSet<AnimationSetKey>) {
         self.find(|fragment, _| {
-            if let Some(tag) = fragment.tag().as_ref() {
-                set.remove(&tag.node());
-            }
+            let (node, pseudo) = match fragment.tag() {
```

Comment:
nit: can you use `match fragment.tag()? { Tag::Node(...) => { .. } }`?

Reply:
Sure. This is a lot nicer.